webview/src/experiments/components/table/styles.module.scss

```diff
@@ -1,11 +1,11 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
`table/styles.module.scss` had a lot of errors with `no-descending-specificity`. I disabled this file rule for now since it will take some time to take care of. 

```
dvc-vscode-webview:lint:css: src/experiments/components/table/styles.module.scss
dvc-vscode-webview:lint:css:  222:1  ✖  Expected selector ".experimentsTh" to come before selector ".experimentsTh:first-child"                                                                               no-descending-specificity
dvc-vscode-webview:lint:css:  284:3  ✖  Expected selector ".firstLevelHeaderCell .paramHeaderCell" to come before selector ".headRow:last-child .paramHeaderCell"                                             no-descending-specificity
dvc-vscode-webview:lint:css:  289:3  ✖  Expected selector ".firstLevelHeaderCell .metricHeaderCell" to come before selector ".headRow:last-child .metricHeaderCell"                                           no-descending-specificity
dvc-vscode-webview:lint:css:  294:3  ✖  Expected selector ".firstLevelHeaderCell .depHeaderCell" to come before selector ".headRow:last-child .depHeaderCell"                                                 no-descending-specificity
dvc-vscode-webview:lint:css:  330:3  ✖  Expected selector ".iconMenu ul[role='menu']" to come before selector ".dropTargetHeaderCell .iconMenu ul[role='menu']"                                               no-descending-specificity
dvc-vscode-webview:lint:css:  352:3  ✖  Expected selector ".moveToRight ul[role='menu']" to come before selector ".dropTargetHeaderCell .iconMenu ul[role='menu']"                                            no-descending-specificity
dvc-vscode-webview:lint:css:  507:1  ✖  Expected selector ".experimentsTd" to come before selector ".experimentsTd:first-child"                                                                               no-descending-specificity
dvc-vscode-webview:lint:css:  514:3  ✖  Expected selector ".experimentsTd:not(.experimentCell)::before" to come before selector ".bodyRow:hover .experimentsTd:not(.experimentCell):hover::before"            no-descending-specificity
dvc-vscode-webview:lint:css:  514:3  ✖  Expected selector ".experimentsTd:not(.experimentCell)::before" to come before selector ".bodyRow:hover .experimentsTd:hover + .experimentsTd::before"                no-descending-specificity
dvc-vscode-webview:lint:css:  514:3  ✖  Expected selector ".experimentsTd:not(.experimentCell)::before" to come before selector ".rowSelected:hover .experimentsTd:not(.experimentCell):hover::before"        no-descending-specificity
dvc-vscode-webview:lint:css:  514:3  ✖  Expected selector ".experimentsTd:not(.experimentCell)::before" to come before selector ".rowSelected:hover .experimentsTd:hover + .experimentsTd::before"            no-descending-specificity
dvc-vscode-webview:lint:css:  536:3  ✖  Expected selector ".experimentsTd:first-child" to come before selector ".rowSelected .experimentsTd:not(.experimentCell)"                                             no-descending-specificity
dvc-vscode-webview:lint:css:  536:3  ✖  Expected selector ".experimentsTd:first-child" to come before selector ".bodyRow:not(.rowSelected):hover .experimentsTd"                                              no-descending-specificity
dvc-vscode-webview:lint:css:  536:3  ✖  Expected selector ".experimentsTd:first-child" to come before selector ".bodyRow:not(.rowSelected):hover .experimentsTd:not(.experimentCell)"                         no-descending-specificity
dvc-vscode-webview:lint:css:  542:3  ✖  Expected selector ".experimentsTd:last-child" to come before selector ".rowSelected .experimentsTd:not(.experimentCell)"                                              no-descending-specificity
dvc-vscode-webview:lint:css:  542:3  ✖  Expected selector ".experimentsTd:last-child" to come before selector ".bodyRow:not(.rowSelected):hover .experimentsTd"                                               no-descending-specificity
dvc-vscode-webview:lint:css:  542:3  ✖  Expected selector ".experimentsTd:last-child" to come before selector ".bodyRow:not(.rowSelected):hover .experimentsTd:not(.experimentCell)"                          no-descending-specificity
dvc-vscode-webview:lint:css:  549:1  ✖  Expected selector ".innerCell" to come before selector ".experimentsTd:first-child .innerCell"                                                                        no-descending-specificity
dvc-vscode-webview:lint:css:  549:1  ✖  Expected selector ".innerCell" to come before selector ".experimentsTd:last-child .innerCell"                                                                         no-descending-specificity
dvc-vscode-webview:lint:css:  567:3  ✖  Expected selector ".experimentCell .innerCell" to come before selector ".experimentsTd:first-child .innerCell"                                                        no-descending-specificity
dvc-vscode-webview:lint:css:  567:3  ✖  Expected selector ".experimentCell .innerCell" to come before selector ".experimentsTd:last-child .innerCell"                                                         no-descending-specificity
dvc-vscode-webview:lint:css:  578:1  ✖  Expected selector ".rowActions" to come before selector ".workspaceRowGroup .rowActions"                                                                              no-descending-specificity
dvc-vscode-webview:lint:css:  619:3  ✖  Expected selector ".starSwitch svg" to come before selector ".iconMenu ul[role='menu'] button svg"                                                                    no-descending-specificity
dvc-vscode-webview:lint:css:  622:5  ✖  Expected selector ".rowSelected .starSwitch svg" to come before selector ".iconMenu ul[role='menu'] button svg"                                                       no-descending-specificity
dvc-vscode-webview:lint:css:  627:3  ✖  Expected selector ".starSwitch[aria-checked='true'] svg" to come before selector ".iconMenu ul[role='menu'] button svg"                                               no-descending-specificity
dvc-vscode-webview:lint:css:  748:3  ✖  Expected selector ".experimentCellText > *" to come before selector ".experimentsTable.withExpColumnShadow .experimentsTr > *:first-child"                            no-descending-specificity
dvc-vscode-webview:lint:css:  759:3  ✖  Expected selector ".experimentCellSecondaryName > *" to come before selector ".experimentsTable.withExpColumnShadow .experimentsTr > *:first-child"                   no-descending-specificity
dvc-vscode-webview:lint:css:  763:3  ✖  Expected selector ".experimentCellSecondaryName svg" to come before selector ".iconMenu ul[role='menu'] button svg"                                                   no-descending-specificity
dvc-vscode-webview:lint:css:  763:3  ✖  Expected selector ".experimentCellSecondaryName svg" to come before selector ".rowSelected .starSwitch svg"                                                           no-descending-specificity
dvc-vscode-webview:lint:css:  763:3  ✖  Expected selector ".experimentCellSecondaryName svg" to come before selector ".starSwitch[aria-checked='true'] svg"                                                   no-descending-specificity
dvc-vscode-webview:lint:css:  784:3  ✖  Expected selector ".experimentCellSecondaryNameTooltip *:first-child" to come before selector ".experimentsTable.withExpColumnShadow .experimentsTr > *:first-child"  no-descending-specificity
dvc-vscode-webview:lint:css:  794:5  ✖  Expected selector ".experimentCellSecondaryNameTooltip .sha svg" to come before selector ".iconMenu ul[role='menu'] button svg"                                       no-descending-specificity
dvc-vscode-webview:lint:css:  814:1  ✖  Expected selector ".timestampInnerCell" to come before selector ".workspaceRowGroup .timestampInnerCell"                                                              no-descending-specificity
dvc-vscode-webview:lint:css:  867:3  ✖  Expected selector ".commitsAndBranchesNavButton:disabled" to come before selector ".commitsAndBranchesNavButton:hover:not(:disabled)"                                 no-descending-specificity
dvc-vscode-webview:lint:css: 
dvc-vscode-webview:lint:css: 34 problems (34 errors, 0 warnings)
```

Personally, I think taking care of these will lower the chance of specificity issues and clean things up more but we could also consider turning off this rule entirely if we think it's too strict to upkeep.

Reply:
If it is a useful rule we should fix.

Reply:
I think it would be really good to fix these. It might be too big for this PR, but a follow up just for this would be great.

Reply:
Glad we're all in agreement! Will plan to do this in a followup.